### PR TITLE
Provides some basic charts and repors for sirius frameworks.

### PR DESCRIPTION
**Pull request by andyHa**
This also contains some improvements of the DataExplorer itself.

Requires:
* https://github.com/scireum/sirius-db/pull/521

grafik

### CODE REVIEW

File: src/main/java/sirius/biz/jobs/BasicJobFactory.java [hunk 1]
```diff
@@ -265,15 +270,10 @@ protected void setupTaskContext() {
...
-     * <p>
-     * You cannot override this method, because it should behave consistently with {@link #getRequiredPermissions()}.
-     * Please add all required permissions there.
-     */
-    protected final void checkPermissions() {
-        UserInfo currentUser = UserContext.getCurrentUser();
-        getRequiredPermissions().forEach(currentUser::assertPermission);
+    protected final void enforceAccessibility() {
+        if (!isAccessibleToCurrentUser()) {
+            throw Exceptions.createHandled().withDirectMessage(NLS.get("BasicJobFactory.unauthorized")).handle();
```
Comment (sabieber) [inline src/main/java/sirius/biz/jobs/BasicJobFactory.java:275]: Is this translation Key already present?

File: src/main/java/sirius/biz/web/Action.java [hunk 2]
```diff
@@ -20,10 +20,10 @@
  */
 public class Action {
 
-    private static final String[] COLOR_CLASSES = {"text-sirius-cyan",
-                                                   "text-sirius-green",
+    private static final String[] COLOR_CLASSES = {"text-sirius-green",
```
Comment (sabieber) [inline src/main/java/sirius/biz/web/Action.java:23]: Commit message says the color palette was extended but no new colors were added

File: src/main/java/sirius/biz/tenants/metrics/AvgActivityPerTenantChart.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.function.Consumer;
+
+/**
+ * Provides a chart showing the number of {@link TenantMetricComputer#METRIC_AVG_ACTIVITY} for a tenant.
+ */
+@Register(framework = Tenants.FRAMEWORK_TENANTS)
+public class AvgActivityPerTenantChart extends TimeSeriesChartFactory<Tenant<?>> {
```
Comment (sabieber) [inline src/main/java/sirius/biz/tenants/metrics/AvgActivityPerTenantChart.java:34]: Optional: `AverageActivityPerTenantChart`, also for the other Avg Charts

**Review by sabieber — APPROVED** [review]

File: src/main/java/sirius/biz/web/Action.java (hunk 2 above)
Comment (andyHa) [reply src/main/java/sirius/biz/web/Action.java:23]: Updated would have been a better verb ;-)

**Review by sabieber — APPROVED** [review]

**Review by idlira — APPROVED** [review]

**Pull request merged**